App/Lettuce/Sources/Lettuce/MessageCenter/Message/GET/Message.swift

```diff
@@ -0,0 +1,68 @@
+// system
+import Foundation
+import HTTPTypes
+
+// local
+import Onion
+
+/// https://developer.allegro.pl/documentation#operation/getMessageGET
+public struct Message: Codable, Equatable, ContentType {
+    public let id: String
+    
+    public enum Status: String, Codable {
+        case verifying = "VERIFYING"
+        case blocked = "BLOCKED"
+        case delivered = "DELIVERED"
+        case interacting = "INTERACTING"
+        case dismissed = "DISMISSED"
+    }
+    public let status: Status.RawValue
```

Comment:
How this will work? Will accept a `String` or an instance of `Message.Status` type?

Reply:
Fixed, String